assets/styles/main.scss

```diff
@@ -186,6 +186,17 @@ body.hide-sabbath:not(.has-sidebar--posts) {
   }
 }
 
+// Guidepost styles
+.wp-block-sbb-guidepost {
+  margin-left: 40px;
```

Comment:
@kelscahill Can you add a left margin for nested items? Right now, Any level below the `h2` level just shows with a left margin of `0`.

## Edit Screen
![2019-11-18 at 1 52 PM](https://user-images.githubusercontent.com/132864/69080789-9f7de480-0a0a-11ea-9a1e-72c7b8cc6d4a.png)

## Front End
![2019-11-18 at 1 51 PM](https://user-images.githubusercontent.com/132864/69080760-92f98c00-0a0a-11ea-9ffb-3083f94ed24e.png)